File: Account.cs
 public abstract class Account
{
    protected static long accountID = 0;
    protected long phonenumber;
    protected string email;

    public Account(long phonenum, string em)
    {
        accountID = accountID++;
        phonenumber = phonenum;
        email = em;
    }

    public long getAccount()
    {
        return accountID;
    }
    public long getPhone()
    {
        return phonenumber;
    }

    public string getEmail()
    {
        return email;
    }
    public long setPhone(long p)
    {
        phonenumber = p;
        return phonenumber;
    }
    public string setEmail(string e)
    {
        email = e;
        return email;
    }
}

public class Customer : Account
{
    public Customer(long phonenum, string em) : base( phonenum, em) 
    {
        accountID = accountID++;
        phonenumber = phonenum;
        email = em;
    }
}
public class Employee : Account
{
    public Employee( long phonenum, string em) : base( phonenum, em)
    {
        accountID = accountID++;
        phonenumber = phonenum;
        email = em;
    }
}

File: user/Account.java
package user.user;

public class Account {
    protected static long accountID = 0;
    protected static long phoneNumber;
    protected static String email;
    protected static String name;

    public Account(long phoneNumber, String email, String name) {
        this.accountID++;
        this.phoneNumber = phoneNumber;
        this.email = email;
        this.name = name;
    }
    public static String getName(){return name;}
    public long getAccount() {
        return accountID;
    }

    public long getPhoneNumber() {
        return phoneNumber;
    }

    public String getEmail() {
        return email;
    }

    public void setPhoneNumber(long phoneNumber) {
        this.phoneNumber = phoneNumber;
    }

    public void setEmail(String email) {
        this.email = email;
    }
}
